blocks/code/src/dev.tsx

```diff
@@ -7,22 +7,34 @@ import { createRoot } from "react-dom/client";
 
 import packageJSON from "../package.json";
 import Component from "./index";
+import { RootEntity } from "./types.gen";
 
 const node = document.getElementById("app");
 
 /**
  * @type {{content: string; language: import("./utils").LanguageType;}}
  */
-const initialData = {
-  content: 'var foo = "bar";',
-  language: "javascript",
+const initialData: RootEntity = {
+  metadata: {
+    entityTypeId: "https://alpha.hash.ai/@ciaran/types/entity-type/code/v/1",
+    editionId: {
+      baseId: "entity-code",
+      versionId: "1",
```

Comment:
Does this not cause problems?

Reply:
(I would have expected a version to be a timestamp, or this to be `recordId`)

Reply:
No, because it's only used for getting a root from the subgraph, and both the vertices and the roots in this mock data are using the same `versionId`. Nothing is actually using version history. We'll be prompted to update this when we make the entity edition ids consistent between BP and HASH

Reply:
If you run `yarn dev`, what happens when you actually load it into HASH though? Does the demo data get ignored?

Reply:
Yes, because HASH does not use this file. This is the entry point for the development server. HASH uses the bundled source of `app.tsx`

Reply:
Forgot about how it loaded them from a URL (wasn't sure if it was the bundled source or if it just sorta embedded the live view 😅)